app/src/main/res/layout/tx_detail_emoji_id_summary.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We already have an emoji id summary view - the design looks the same, why not reuse it?